TokenManager.js

```diff
@@ -312,15 +316,17 @@
                         return Promise.reject("invalid_token");
                     }
                 })
-                .then((request) => fetch(request.url, request.options))
+                .then((request) => {
+                    return fetch(request.url, request.options);})
```

Comment:
Why make this change?

Reply:
I'll switch back to implicit returns - it was easier to read with the and know if the return was needed with returns on a different line